storage/tests/unit/test_bucket.py

```diff
@@ -266,16 +266,16 @@ def test_from_api_repr_w_enabled(self):
         config = klass.from_api_repr(resource, bucket)
 
         self.assertIs(config.bucket, bucket)
-        self.assertTrue(config.bucket_policy_only_enabled)
-        self.assertEqual(config.bucket_policy_only_locked_time, now)
+        self.assertTrue(config.uniform_bucket_level_access_enabled)
+        self.assertEqual(config.uniform_bucket_level_access_locked_time, now)
```

Comment:
Preserve the assertions about the deprecated properties here, too.

Also, do we need a separate testcase for a resource which contains only a populated `bucketPolicyOnly` mapping? I.e., can we guarantee that the back-end won't return such a resource?